engine/src/json.rs

```diff
@@ -90,10 +90,13 @@ impl JsonValue {
     pub fn parse_u8(v: &JsonValue) -> Result<u8, JsonError> {
         match v {
             JsonValue::U64(n) => {
-                if *n > u8::MAX as u64 {
+                if *n > u64::from(u8::MAX) {
                     Err(JsonError::OutOfRange(JsonOutOfRangeError::OutOfRangeU8))
                 } else {
-                    Ok(*n as u8)
+                    match u8::try_from(*n) {
```

Comment:
This is redundant after the `if` check above. Probably better to simply replace the `if` check with this `match`.

Reply:
Good catch.

Reply:
Improved in https://github.com/aurora-is-near/aurora-engine/pull/561/commits/926634beb14066bdfef0e0f7342cb3b5fc2bfcb7